src/porepy/compositional/compositional_mixins.py

```diff
@@ -1128,17 +1174,14 @@ def get_phase_configuration(
             :class:`~porepy.compositional.base.Phase`.
 
         """
-        raise CompositionalModellingError(
-            "Call to mixin method. Configure phases by overriding this method."
-        )
+        return [(AbstractEoS(components), PhysicalState.liquid, "liquid")]
```

Comment:
The default behavior of get_phase_configuration was also changed to model a 1-phase fluid with a liquid-like phase.
The AbstractEoS was modified for this purpose, not being abstract anymore but playing the role of the "mathematician's" EoS, i.e. a dummy doing nothing for models using heuristic fluid properties (constitutive_laws.py)

Reply:
Can be resolved for my sake